From 364b90e91c4ce7465dc13bb792098f9ab238d053 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Thu, 7 Nov 2019 09:55:12 +0700
Subject: [PATCH 308/326] arm64: defconfig: Enable RENESAS_MFIS_ECC

Enable RENESAS_MFIS_ECC to support DRAM ECC error interrupt.

Signed-off-by: Binh Nguyen <binh.nguyen.uw@renesas.com>
Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index a37af4c..a056780 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -364,3 +364,4 @@ CONFIG_DEVFREQ_GOV_POWERSAVE=y
 CONFIG_DEVFREQ_GOV_USERSPACE=y
 CONFIG_DEVFREQ_GOV_PASSIVE=y
 CONFIG_E1000E=y
+CONFIG_RENESAS_MFIS_ECC=y
-- 
1.9.1

